src/masternodes/mn_checks.cpp

```diff
@@ -1621,19 +1627,19 @@ Res ApplyCloseOrderTx(CCustomCSView & mnview, CCoinsViewCache const & coins, CTr
         return Res::Err("%s: deserialization failed: excess %d bytes", __func__, ss.size());
     }
     
-    closeorder.creationTx=tx.GetHash();
+    closeorder.creationTx = tx.GetHash();
     closeorder.creationHeight = height;
 
     std::unique_ptr<COrderImplemetation> order;
-    if (!(order=mnview.GetOrderByCreationTx(closeorder.orderTx))) {
+    if (!(order = mnview.GetOrderByCreationTx(closeorder.orderTx))) {
         return Res::Err("order with creation tx %s does not exists!", closeorder.orderTx.GetHex());
     }
     if (!order->closeTx.IsNull()) {
         return Res::Err("order with creation tx %s is already closed!", closeorder.orderTx.GetHex());
     }
 
-    order->closeTx=closeorder.creationTx;
-    order->closeHeight=closeorder.creationHeight;
+    order->closeTx = closeorder.creationTx;
+    order->closeHeight = closeorder.creationHeight;
```

Comment:
These are unsaved, no? `Order` should be updated in db.

Reply:
On line 1654 this is called `res = mnview.CloseOrderTx(*order);` which updates the DB for order.

Reply:
That's right, fine.